TypeScript のwarningが出る

# 概要
<!-- いつ/どこで/どんな不具合が発生しているのか -->
```
frontend_1  | 
frontend_1  | ERROR in /app/src/app.tsx
frontend_1  | ./src/app.tsx 4:16-31
frontend_1  | [tsl] ERROR in /app/src/app.tsx(4,17)
frontend_1  |       TS2307: Cannot find module 'pages/sekigae' or its corresponding type declarations.
frontend_1  | 
frontend_1  | ERROR in /app/src/components/atom/askButton.tsx
frontend_1  | ./src/components/atom/askButton.tsx 6:29-33
frontend_1  | [tsl] ERROR in /app/src/components/atom/askButton.tsx(6,30)
frontend_1  |       TS6133: 'prop' is declared but its value is never read.
frontend_1  |  @ ./src/components/organism/menuBar.tsx 6:0-50 82:19-28
frontend_1  |  @ ./src/pages/sekigae.tsx 6:0-50 327:21-28
frontend_1  |  @ ./src/app.tsx 3:0-32 7:59-62
frontend_1  | 
frontend_1  | ERROR in /app/src/components/atom/startButton.tsx
frontend_1  | ./src/components/atom/startButton.tsx 6:31-35
frontend_1  | [tsl] ERROR in /app/src/components/atom/startButton.tsx(6,32)
frontend_1  |       TS6133: 'prop' is declared but its value is never read.
frontend_1  |  @ ./src/pages/sekigae.tsx 5:0-54 357:21-32 451:21-32
frontend_1  |  @ ./src/app.tsx 3:0-32 7:59-62
frontend_1  | 
frontend_1  | ERROR in /app/src/components/organism/menuBar.tsx
frontend_1  | ./src/components/organism/menuBar.tsx 6:24-53
frontend_1  | [tsl] ERROR in /app/src/components/organism/menuBar.tsx(6,25)
frontend_1  |       TS2307: Cannot find module 'components/atom/startButton' or its corresponding type declarations.
frontend_1  |  @ ./src/pages/sekigae.tsx 6:0-50 327:21-28
frontend_1  |  @ ./src/app.tsx 3:0-32 7:59-62
frontend_1  | 
frontend_1  | ERROR in /app/src/components/organism/menuBar.tsx
frontend_1  | ./src/components/organism/menuBar.tsx 7:22-49
frontend_1  | [tsl] ERROR in /app/src/components/organism/menuBar.tsx(7,23)
frontend_1  |       TS2307: Cannot find module 'components/atom/askButton' or its corresponding type declarations.
frontend_1  |  @ ./src/pages/sekigae.tsx 6:0-50 327:21-28
frontend_1  |  @ ./src/app.tsx 3:0-32 7:59-62
frontend_1  | 
frontend_1  | ERROR in /app/src/pages/sekigae.tsx
frontend_1  | ./src/pages/sekigae.tsx 6:24-53
frontend_1  | [tsl] ERROR in /app/src/pages/sekigae.tsx(6,25)
frontend_1  |       TS2307: Cannot find module 'components/atom/startButton' or its corresponding type declarations.
frontend_1  |  @ ./src/app.tsx 3:0-32 7:59-62
frontend_1  | 
frontend_1  | ERROR in /app/src/pages/sekigae.tsx
frontend_1  | ./src/pages/sekigae.tsx 7:20-49
frontend_1  | [tsl] ERROR in /app/src/pages/sekigae.tsx(7,21)
frontend_1  |       TS2307: Cannot find module 'components/organism/menuBar' or its corresponding type declarations.
frontend_1  |  @ ./src/app.tsx 3:0-32 7:59-62
frontend_1  | 
frontend_1  | ERROR in /app/src/pages/sekigae.tsx
frontend_1  | ./src/pages/sekigae.tsx 9:18-40
frontend_1  | [tsl] ERROR in /app/src/pages/sekigae.tsx(9,19)
frontend_1  |       TS2307: Cannot find module 'assets/classroom.png' or its corresponding type declarations.
frontend_1  |  @ ./src/app.tsx 3:0-32 7:59-62
frontend_1  | 
frontend_1  | ERROR in /app/src/pages/sekigae.tsx
frontend_1  | ./src/pages/sekigae.tsx 10:20-40
frontend_1  | [tsl] ERROR in /app/src/pages/sekigae.tsx(10,21)
frontend_1  |       TS2307: Cannot find module 'assets/workman.png' or its corresponding type declarations.
frontend_1  |  @ ./src/app.tsx 3:0-32 7:59-62
frontend_1  | 
frontend_1  | ERROR in /app/src/pages/sekigae.tsx
frontend_1  | ./src/pages/sekigae.tsx 11:17-42
frontend_1  | [tsl] ERROR in /app/src/pages/sekigae.tsx(11,18)
frontend_1  |       TS2307: Cannot find module 'assets/YounGeekLogo.png' or its corresponding type declarations.
frontend_1  |  @ ./src/app.tsx 3:0-32 7:59-62
frontend_1  | 
frontend_1  | 10 errors have detailed information that is not shown.
frontend_1  | Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.

```


# 再現手順
<!-- どうやって再現するのか -->
1. build + up　する


# 修正の背景
<!-- 修正が必要な理由 -->
- 


# 修正後の要件
<!-- 修正後にどのような挙動が期待されるか -->
- 


# 原因
<!-- 不具合の原因を書く -->
<!-- 原因がはっきりと分からない場合でも、「ここが怪しそう」などあれば書いておく。 -->
- おそらくモジュールに型書いてあげれば良いんじゃないかなと思ってます


# 修正方針
<!-- どうやって修正するのか -->
- 

# テスト項目
<!-- 不具合が修正されたかどうかを確認する方法 -->
- 

<!-- 全ては埋めなくても良いが、必要な情報をわかりやすく書く。 -->

--- a/frontend/.babelrc
+++ b/frontend/.babelrc
@@ -3,5 +3,8 @@
       "@babel/preset-env",
       "@babel/preset-react",
       "@emotion/babel-preset-css-prop"
+  ],
+  "plugins": [
+    ["module-resolver"]
   ]
 }

--- a/frontend/package.json
+++ b/frontend/package.json
@@ -32,6 +32,7 @@
     "@emotion/babel-preset-css-prop": "^11.2.0",
     "@emotion/core": "^11.0.0",
     "@emotion/react": "^11.4.1",
+    "babel-plugin-module-resolver": "^4.1.0",
     "file-loader": "^6.2.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2"

--- a/frontend/src/app.tsx
+++ b/frontend/src/app.tsx
@@ -3,7 +3,7 @@ import ReactDOM from 'react-dom'
 
 import Top from 'pages/sekigae'
 
-const App: React.FC = () => {
+const App: React.FC<{}> = () => {
     return(
         <>
             <Top />

--- a/frontend/src/components/atom/askButton.tsx
+++ b/frontend/src/components/atom/askButton.tsx
@@ -1,9 +1,8 @@
 /** @jsx jsx */
-import * as React from "react"
 import { jsx } from "@emotion/react"
 import { css } from "@emotion/react"
 
-const AskButton: React.FC = (prop) => {
+const AskButton = () => {
   return (
     <div
       css={css({

--- a/frontend/src/components/atom/startButton.tsx
+++ b/frontend/src/components/atom/startButton.tsx
@@ -1,9 +1,8 @@
 /** @jsx jsx */
-import * as React from "react"
 import { jsx } from "@emotion/react"
 import { css } from "@emotion/react"
 
-const StartButton: React.FC = (prop) => {
+const StartButton = () => {
   return (
     <div
       css={css({

--- a/frontend/src/components/organism/menuBar.tsx
+++ b/frontend/src/components/organism/menuBar.tsx
@@ -6,7 +6,7 @@ import { css } from "@emotion/react"
 import StartButton from "components/atom/startButton"
 import AskButton from "components/atom/askButton"
 
-const MenuBar: React.FC = () => {
+const MenuBar: React.FC<{}> = () => {
   return (
     <div
       css={css({

--- a/frontend/src/pages/sekigae.tsx
+++ b/frontend/src/pages/sekigae.tsx
@@ -14,7 +14,7 @@ const primary = "#F9F871"
 const secondary = "#60A8B0"
 const tertiary = "#555555"
 
-const Top: React.FC = () => {
+const Top: React.FC<{}> = () => {
   const heading = css({
     color: primary,
     fontSize: "4vw",

--- a/frontend/tsconfig.json
+++ b/frontend/tsconfig.json
@@ -2,6 +2,7 @@
     "compilerOptions": {
         "target": "esnext",
         "jsx": "preserve",
+        "baseUrl": "src",
         "typeRoots": [
             "node_modules/@types",
             "types"

--- a/frontend/tsconfig.json
+++ b/frontend/tsconfig.json
@@ -2,7 +2,10 @@
     "compilerOptions": {
         "target": "esnext",
         "jsx": "preserve",
-
+        "typeRoots": [
+            "node_modules/@types",
+            "types"
+        ],
         "forceConsistentCasingInFileNames": true,
         "strict": true,
         "noImplicitAny": true,
@@ -15,6 +18,10 @@
         "isolatedModules": true,
         "allowSyntheticDefaultImports": true,
     },
-    "include": ["src/**/*.ts", "src/**/*.tsx"],
+    "include": [
+        "src/**/*.ts", 
+        "src/**/*.tsx",
+        "types/**/*"
+    ],
     "exclude": ["node_modules"]
 }

--- a/frontend/types/images.d.ts
+++ b/frontend/types/images.d.ts
@@ -1,0 +1,3 @@
+declare module "*.jpg"
+declare module "*.png"
+declare module "*.svg"

